src/components/forms/PlanForm/tests/index.test.tsx

```diff
@@ -4,6 +4,7 @@ import { mount, ReactWrapper, shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import { cloneDeep } from 'lodash';
 import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
Decided to fix a test error message that PlanForm should be wrapped within a router component